website/src/_posts/2021-08-30.31.md

```diff
@@ -0,0 +1,250 @@
+
+**So today we launch Uppy 2.0 🎊**
+
+With 2.0, we have made Uppy 25% smaller, up to 1000X faster, and more intuitive to work with. The details on these claims are further down in the post. 2.0 also marks for us another milestone. Since 0.1 we have had the tagline:
+
+> The next open source file uploader for webbrowsers
+
+Between 1.0 and 2.0, we’ve seen adoption ramp up incredibly steeply and now with over [24,000 stargazers on GitHub](https://github.com/transloadit/uppy/stargazers) we can safely say, Uppy has become the #1 file uploader in the world 🌍
+
+![Uppy 2.0 UI with files](/images/blog/2.0/uppy-ui-with-files.jpg)
+
+## Table of Contents
+
+- [Highlights since 1.0](#Highlights-since-1-0)
+- [Smaller bundles](#Smaller-bundles)
+- [Faster](#Faster)
+- [Preact X and upgraded dependencies](#Preact-X-and-upgraded-dependencies)
+- [`Plugin` is replaced with `BasePlugin` and `UIPlugin`](#Plugin-is-replaced-with-BasePlugin-and-UIPlugin)
+- [Strict TypeScript types by default](#Strict-TypeScript-types-by-default)
+- [Batch pre-signing URL's for AWS S3 Multipart](#Batch-pre-signing-URL’s-for-AWS-S3-Multipart)
+- [And more](#And-more)
+- [What future remains for 1.0](#What-future-remains-for-1-0)
+- [That's it](#That’s-it)
+
+## Highlights since 1.0
+
+A lot of things have happened since 1.0. In case you have missed some of our 1.x releases, here is a small overview. We have added:
+
+- Official integrations for [Vue](/blog/2020/11/1.23/) ([3](/blog/2021/04/1.27/)), [Angular](/blog/2021/07/1.30/), and [Svelte](/blog/2020/12/1.24/)
+- [Hooks](http://localhost:4000/blog/2020/12/1.24/) for the React integration
+- Support for picking files from [Box](/blog/2020/11/1.23/), [Unsplash](/blog/2020/10/1.22/), [Facebook](/blog/2020/04/1.10/), [OneDrive](/blog/2020/04/1.10/) 
+- [“Ghost” files](/blog/2021/05/1.29/), as part of a revamped Golden Retriever Plugin, which makes recovering lost files more intuitive
+- Support for 25 more languages (to a [total of 37](/docs/locales/#List-of-locale-packs))
+- [Dark mode](/blog/2020/04/1.13/)
+- [Image Editor](/blog/2020/07/1.18-image-editor/)
+- Four new core team members: [@ajkachnic](https://github.com/ajkachnic), [@mifi](https://github.com/mifi), [@aduh95](https://github.com/aduh95), and [@murderlon](https://github.com/murderlon).
+
+## Smaller bundles
+
+With 2.0, following Microsoft, we are dropping support for IE11. As a result, we are able to remove all built-in polyfills, and the new bundle size is 25% smaller! If you need your app to support older browsers (such as IE11), you should add the following polyfills to your bundle:
```

Comment:
I wonder if we should link to the relevant guide instead, in case the list of polyfills is amended in a future semver-minor version. wdyt?

Reply:
I'm personally okay with leaving that to the changelog. Future versions could reference changes in polyfills. We don't officially support IE as of 2.0.0 so if people decide to use future releases it can be expected not to immediately work. Besides, maintaining a guide with polyfills would mean testing it with polyfills mentioned, which is essentially capacity spend on maintenance again.

Reply:
Maybe we can rephrase this so it doesn't imply its an exhaustive list
```suggestion
With 2.0, following Microsoft, we are dropping support for IE11. As a result, we are able to remove all built-in polyfills, and the new bundle size is 25% smaller! If you need your app to support older browsers (such as IE11), you may need to add the following polyfills to your bundle:
```